lib/css/components/sidebar-widgets.less

```diff
@@ -0,0 +1,245 @@
+                    height: calc(100% + var(--_sw-content-spacing-inner));
+                    margin-top: calc(var(--_sw-content-spacing-inner) / 2 * -1);
+                    margin-left: calc(var(--_sw-content-px) * -1 - 1px); // the orange selection indicator overlaps with the widget border
+                    border-left-width: 3px;
+                    border-left-style: solid;
+                    border-left-color: var(--theme-primary-color);
+                }
+
+                a,
+                a:visited { // TODO: this isn't the best way to go about this. There should be a "is current" highlight without font modification for more complex cases
+                    color: inherit;
+                }
+            }
+        }
+
+        margin: var(--_sw-content-spacing-inner) 0;
+    }
+
+    & &--subnav {
+        li {
+            #stacks-internals #bullet-arrow(var(--black-100));
+
+            &[aria-current="true"],
+            &[aria-current="page"] {
+                #stacks-internals #bullet-arrow(var(--theme-primary-color));
+                color: var(--black);
+                font-weight: bold;
+
+                a,
+                a:visited { // TODO: see above
+                    color: inherit;
+                }
+            }
+
+            margin-top: var(--su-static12);
+            padding-left: var(--su-static16);
+            background-position: 0 calc((1.2em - 10px) / 2);
+            background-repeat: no-repeat;
+            background-size: auto 10px;
+        }
+
+        list-style-type: none;
+        padding-left: 0;
+        margin-left: var(--su8);
+    }
+
+    & table.s-sidebarwidget--content.s-sidebarwidget__items {
+        tr.s-sidebarwidget--item {
+            display: table-row;
+            td {
+                padding: 0;
+            }
+        }
+
+        padding: calc(var(--_sw-content-spacing-outer) - var(--_sw-content-spacing-inner)) calc(var(--_sw-content-px) - var(--_sw-content-spacing-inner));
+        border-collapse: separate;
+        border-spacing: var(--_sw-content-spacing-inner);
+    }
+
+    &:not(.s-anchors) a:not(.button):not(.s-btn):not(.post-tag):not(&--action):not(.s-user-card--link) {
+        &,
+        &:visited {
+            color: var(--black-500);
+        }
+    }
+
+    &:before { // [1]
+        content: '';
+        display: block;
+        margin-top: -1px;
+    }
+
+    &:after { // [2]
+        content: '';
+        position: absolute;
+        top: -1px;
+        right: -1px;
+        left: -1px;
+        // TODO: this makes no sense. revisit.
+        height: calc(2px + var(--br-sm)); // we need 2px border + 2px border radius to have the correct corner shape
+        border-radius: var(--br-sm);
+        pointer-events: none; // if the top item is clickable, then we don't want to prevent clicking the top 2 pixels
+        clip-path: polygon(0% 0%, 100% 0%, 100% 50%, 0% 50%); // [3]
+        border: 1px solid var(--_sw-bc);
```

Comment:
I'd suggest creating a new custom property named `--_sw-after-bc`. If it's always the same as `--_sw- bc`, then we can define it as `--_sw-after-bc: var(--_sw-bc)`.

Basically, I want to ensure that a contributor can quickly parse what _might_ change from just a glance at the variables defined at the start of the file.

Reply:
Will address this.